packages/zowe-explorer/src/extension.ts

```diff
@@ -66,16 +66,13 @@ export async function activate(context: vscode.ExtensionContext): Promise<ZoweEx
     // Determine the runtime framework to support special behavior for Theia
     globals.defineGlobals(preferencesTempPath);
 
-    // Call cleanTempDir before continuing
-    // this is to handle if the application crashed on a previous execution and
-    // VSC didn't get a chance to call our deactivate to cleanup.
-    await deactivate();
```

Comment:
Oh wow, that was a bad idea. Why not call cleanup directly? What if others would have added more code to the deactiviate function that would have broken activation?

Reply:
@phaumer should I add back a call to cleanup?  calling cleanup during activation is what was causing the deletion of a file right when new window was opened.  With this PR that is not done now but if the new setting is set to false (default) once one of the new windows closes `deleted` will show in other open windows if a ZE file is opened.  If set to true, that doesn't happen since no cleanup is done on temp folder.

Reply:
No, sorry, I was just surprised that we had that code. Should have not commented.